### ISSUE TITLE
Map <option> element to role 'option' when it has a parent <select> element that acts a combobox

### ISSUE THREAD

**Issue by iamrafan** [w3c/html-aam#149]
The [HTML spec for ```<option>```](https://www.w3.org/TR/html5/sec-forms.html#the-option-element) and [HTML-AAM for ```<option>```](https://www.w3.org/TR/html-aam-1.0/#html-element-role-mappings) state that it should always default to role ```option```

But ```<option>```  element's role is returned as ```menuitem``` when used inside a ```<select>``` element that acts as a combobox 

**Code snippets:**
![image](https://user-images.githubusercontent.com/22068434/45652381-4c687480-ba89-11e8-9531-f1c4c3415ea9.png)

![image](https://user-images.githubusercontent.com/22068434/45652455-846fb780-ba89-11e8-8453-e616ea024292.png)

I think the ```<option>``` element here, which is inside ```combobox``` should default to role ```option```

**Comment by scottaohara** [comment]
Hi @iamrafan 

Yes, the browser's accessibility panels expose the role that way, but if you review [the core accessibility api mappings for options](https://www.w3.org./TR/core-aam-1.1/#role-map-option) you'll see that the `option` role is a round up of the different APIs exposing the `option` element as either "ROLE_LIST_ITEM" or "ROLE_MENU_ITEM", the latter being where the `menuitem` role is coming from.

Long story short, HTML-aam already notes to use WAI-ARIA mapping for `option` role, so regardless of what devtools are reporting, this is already in place in the spec.

Closing, but feel free to respond with any followup questions.  Thanks!
